beets/ui/commands.py

```diff
@@ -1479,6 +1480,19 @@ def move_items(lib, dest, query, copy, album, pretend, confirm=False):
             show_path_changes([(obj.path, obj.destination(basedir=dest))
                                for obj in objs])
     else:
+        if export:
+
+            if album:
+                for obj in objs:
+                    for item in obj.items():
+                        # Create necessary ancestry for the copy.
+                        util.mkdirall(item.destination(basedir=dest))
+                        util.copy(item.path, item.destination(basedir=dest))
+            else:
+                for obj in objs:
+                    # Create necessary ancestry for the copy.
+                    util.mkdirall(obj.destination(basedir=dest))
+                    util.copy(obj.path, obj.destination(basedir=dest))
```

Comment:
Do you mean lines 1489&1490 and 1494&1495?? They are not exactly identical, one uses `item` and the other uses `obj`. Do you want me to create a helper function that takes as a parameter either `item` or `obj`?